docs/README.md

```diff
@@ -50,8 +56,9 @@ The subdomain used for the project can be configured via the `modules.local-serv
 
 ## Available Commands
 
-* `composer server start [--xdebug]` - Starts the containers.
-  * If the `--xdebug` option is passed the PHP container will have XDebug enabled. To switch off XDebug run this command again without the `--xdebug` option.
+* `composer server start [--xdebug=<mode>] [--mutagen=(on|off)] [--docker-sync]` - Starts the containers.
+  * `--xdebug` will enable Xdebug. To switch off Xdebug run this command again without the `--xdebug` option.
+  * `--mutagen=(on|off)` will toggle file sharing using [Mutagen](https://mutagen.io/) if installed. Note that the Mutagen Beta version is required.
```

Comment:
This doesn't seems very cli idiomatic. If clean my vendor directory or do a fresh install (or upgrade) I'll need to re-add --mutagen, as the file will be gone from the be vendor dir. Therefor it seems the only logical think to do as a user would be to specify it all the time.

I think it's a lot for expected that passing the flag is ephemeral (just like I assume --xdebug is)-- like I say, I think the config is the right place for any persistent settings

Reply:
If you removed your `vendor` dir it'd remove the `docker-composer.yml` as well. Problem is there isn't really a good place to put files specific to the project that we know are ignored by git. Upgrading wouldn't remove the files if that helps.

This needs to be a flag because the config is a committed file, if one person wants to use it then everyone has to which we don't want.

The difference with the `--xdebug` flag is that even if you didn't specify it when running `composer server stop` or `composer server destroy` it would still be successfully cleaned up. It's not quite as simple when using Mutagen.

It might not be idiomatic but it seems like the best user experience to me. I don't see as being super different to something like `docker login` on the CLI, that changes the behaviour of other commands. I do take your point but I'm not sure what the best option is.

Reply:
I think of we want to take the persistent via cli option, I'd expect it to be another command like "server set-filesysten mutagen" or something along those lines

Reply:
I see yeah. That option might make more sense for the xdebug setting too. Creating and checking for files is the easiest way to do that but perhaps not in the `vendor` directory as you mention. Are you aware of any best practices for storing persistent settings? Maybe in a `.altis` directory in the users home directory.